src/components/TodoList/TodoList.tsx

```diff
@@ -1,100 +1,32 @@
 import React from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem';
 
-export const TodoList: React.FC = () => (
-  <table className="table is-narrow is-fullwidth">
-    <thead>
-      <tr>
-        <th>#</th>
-        <th>
-          <span className="icon">
-            <i className="fas fa-check" />
-          </span>
-        </th>
-        <th>Title</th>
-        <th> </th>
-      </tr>
-    </thead>
+type Props = {
```

Comment:
Use interface prefer.
Type has unique ability but if you don`t need them use interface. It is sampled in scaled and has more interesting features.